### PR TITLE
Add Docs for NNI AutoFeatureEng

### PR DESCRIPTION
**Pull request by JSong-Jia**

### CODE REVIEW

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+including feature engineering, NAS, HPO, etc.
+
+\# \*\*02 Overview of NNI\*\*
+
+NNI (Neural Network Intelligence) is an open source AutoML toolkit from
+Microsoft, to help users design and tune machine learning models, neural network
+architectures, or a complex system’s parameters in an efficient and automatic
+way.
+
+\*\*Address\*\*:[ https://github.com/SpongebBob/tabular_automl_NNI](https://github.com/SpongebBob/tabular_automl_NNI)
```
Comment (scarlett2018) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md]: use the formal url: https://github.com/Microsoft/nni

**Comment by xuehui1991** [comment]
There are some error and can't build the doc successfully.

image

You need to fix it before merge.

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## 03 Details of NNI-AutoFeatureENG
+
+Each new user could do AutoFeatureENG on NNI easily and efficiently:
+
+Firstly, install the require of files
+
+Then, install NNI through pip
+
+![image](https://upload-images.jianshu.io/upload_images/20947594-b2219460951f6a12.jpg?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
```
Comment (xuehui1991) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md]: you could upload the image, too.

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+Conclusion: NNI could offer users some inspirations of design and it is
+a good open source project. I suggest researchers leverage it to accelerate the
+AI research.
+
+Tips: Because the scripts of open source projects are compiled based on
+gcc7, Mac system may encounter problems of gcc (GNU Compiler Collection). The
+solution is as follows:
+
+brew install libomp
```
Comment (xuehui1991) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md]: use 

```
brew install libomp
```
 instead.

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+![image](https://upload-images.jianshu.io/upload_images/20947594-d49c0ead372d4ac0.jpg?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
+
+we can define a cross frequency encoding (value count on cross dims) method on
+columns {"C1",...,"C26"} x {"C1",...,"C26"} in the following way:
+
+![image](https://upload-images.jianshu.io/upload_images/20947594-c58c8d498559c4f0.jpg?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
+
+The purpose of Exploration is to generate new features.You can use
+\*get_next_parameter\* function to get received feature candidates of
```
Comment (xuehui1991) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md]: using `get_next_parameter` instead.

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+including feature engineering, NAS, HPO, etc.
+
+## 02 Overview of NNI
+
+NNI (Neural Network Intelligence) is an open source AutoML toolkit from
+Microsoft, to help users design and tune machine learning models, neural network
+architectures, or a complex system’s parameters in an efficient and automatic
+way.
+
+Address:[ https://github.com/Microsoft/nni](https://github.com/Microsoft/nni)
```
Comment (xuehui1991) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.md]: "Address" --> "Link"

**Pull request merged**